packages/nimble-components/package.json

```diff
@@ -87,6 +91,7 @@
     "@types/d3-selection": "^3.0.0",
     "@types/d3-zoom": "^3.0.0",
     "@types/markdown-it": "^13.0.0",
+    "comlink": "^4.4.1",
```

Comment:
Should comlink be both in here and devDependencies?

Reply:
indeed, it is not needed